.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
we need to update the main .gitignore with .idea

Reply:
Fixed